Description: cross build patch
 This patch enable us to build in cross environment.
 .
 xfireworks (1.3-11) unstable; urgency=low
 .
   [ Yukiharu YABUKI ]
   * Apply dh_auto_build patch (Closes: #913841).
   * Update Standards-Version
 .
   [ Helmut Grohne <helmut@subdivi.de> ]
   * Fix FTCBFS
     * Let dh_auto_bild pass cross tools to make.
     * cross.patch: Use the build architecture compiler for mkconf.c.
Author: Yukiharu YABUKI <yabuki@netfort.gr.jp>
Bug-Debian: https://bugs.debian.org/913841

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <other>, <https://bugs.debian.org/913841>
Bug-Debian: https://bugs.debian.org/913841
Last-Update: 2018-11-29

--- xfireworks-1.3.orig/Makefile
+++ xfireworks-1.3/Makefile
@@ -13,6 +13,7 @@ XLIBDIR = $(X11BASE)/lib
 CFLAGS = -O
 LFLAGS = -lm -lX11
 CC = cc
+CC_FOR_BUILD ?= $(CC)
 CP = cp
 RM = rm -f
 MKDIR = mkdir -p
@@ -66,7 +67,7 @@ xfireworks_conf.h :	xfireworks.conf mkco
 		cat xfireworks.conf | ./mkconf > xfireworks_conf.h
 
 mkconf :	mkconf.c
-		$(CC) $(CPPFLAGS) $(CFLAGS) mkconf.c -o mkconf
+		$(CC_FOR_BUILD) $(CPPFLAGS) $(CFLAGS) mkconf.c -o mkconf
 
 etc.o :		etc.c
 		$(CC) -c etc.c $(CPPFLAGS) $(CFLAGS)
